tests/example-with-output/expected_results.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I found a bug in the script, since the json file with the output isnt being cleared, since I didnt check running multiple exercises in a row. But I have now made a script which clears that json. I am reruning tests.